src/query/plan/operator.cpp

```diff
@@ -4657,23 +4655,12 @@ void CallCustomProcedure(const std::string_view fully_qualified_procedure_name,
   if (memory_limit) {
     SPDLOG_INFO("Running '{}' with memory limit of {}", fully_qualified_procedure_name,
                 utils::GetReadableSize(*memory_limit));
-    utils::LimitedMemoryResource limited_mem(memory, *memory_limit);
-    mgp_memory proc_memory{&limited_mem};
-    MG_ASSERT(result->signature == &proc.results);
-    // TODO: What about cross library boundary exceptions? OMG C++?!
-    proc.cb(&proc_args, &graph, result, &proc_memory);
-    size_t leaked_bytes = limited_mem.GetAllocatedBytes();
-    if (leaked_bytes > 0U) {
-      spdlog::warn("Query procedure '{}' leaked {} *tracked* bytes", fully_qualified_procedure_name, leaked_bytes);
-    }
```

Comment:
This will be added in different PR as part of memory procedure tracking, this one here: https://github.com/memgraph/memgraph/pull/1443